File: Programmers/제일 작은 수 제거하기.java
class Solution {
    public int[] solution(int[] arr) {
        int min = 100000;
        for (int i = 0; i < arr.length; i++) {
            if (arr[i] < min) min = arr[i];
        }
        int[] result = new int[arr.length-1]; int index = 0;
        for (int i = 0; i < arr.length; i++) {
            if (arr[i] != min) {
                result[index] = arr[i];
                index++;
            }
        }
        if (result.length == 0) {
            int[] second = {-1};
            return second;
        }
        return result;
    }
}

File: Programmers/제일 작은 수 제거하기.py
def solution(arr):
    index_check = 0
    min_check = arr[0]
    for i in range(1, len(arr)):
        if arr[i] < min_check:
            min_check = arr[i]
            index_check = i
    del arr[index_check]
    if len(arr) == 0 or (len(arr) == 1 and arr[0] == 10):
        return [-1]
    return arr
